tests/functional/f/fixme.txt

```diff
@@ -1,9 +1,9 @@
-fixme:5:2::"FIXME: beep"
-fixme:11:2::"FIXME: Valid test"
-fixme:14:2::"TODO: Do something with the variables"
-fixme:16:2::"XXX: Fix this later"
-fixme:18:1::"FIXME: no space after hash"
-fixme:20:1::"todo: no space after hash"
-fixme:23:2::"FIXME: this is broken"
-fixme:25:2::"./TODO: find with notes"
-fixme:27:2::"TO make something DO: find with regex"
+fixme:5:1::"FIXME: beep"
+fixme:11:20::"FIXME: Valid test"
+fixme:14:5::"TODO: Do something with the variables"
+fixme:16:18::"XXX: Fix this later"
+fixme:18:5::"FIXME: no space after hash"
+fixme:20:5::"todo: no space after hash"
+fixme:23:9::"FIXME: this is broken"
```

Comment:
Here the new value could be 2 because there is a tabulation (`\t` + a `#`. What do you think ?
```suggestion
fixme:23:2::"FIXME: this is broken"
```

Reply:
I don't think we should count the tabulation  as one column since most editors don't as well. The output message might confuse the user because the column index of the error code will be different for pylint and for the user's editor.

Reply:
But what should the value for a tabulation be then ? 2 spaces, 4 spaces, 8 spaces ? I think the number of spaces displayed depend on the editor's setting.